app/View/Jobs/index.ctp

```diff
@@ -103,16 +101,16 @@
         <table class="table table-striped table-hover table-condensed">
         <tr>
                 <th><?php echo $this->Paginator->sort('id');?></th>
-                <th><?php echo $this->Paginator->sort('date_created');?></th>
-                <th><?php echo $this->Paginator->sort('date_modified');?></th>
-                <th><?php echo $this->Paginator->sort('process_id');?></th>
-                <th><?php echo $this->Paginator->sort('worker');?></th>
-                <th><?php echo $this->Paginator->sort('job_type');?></th>
+                <th><?php echo $this->Paginator->sort('date_created', __('Date created'));?></th>
+                <th><?php echo $this->Paginator->sort('date_modified', __('Date modified'));?></th>
+                <th><?php echo $this->Paginator->sort('process_id', __('Process ID'));?></th>
+                <th><?php echo $this->Paginator->sort('worker', __('Worker'));?></th>
+                <th><?php echo $this->Paginator->sort('job_type', __('Job type'));?></th>
                 <th><?php echo $this->Paginator->sort('job_input', __('Input'));?></th>
                 <th><?php echo $this->Paginator->sort('message');?></th>
-                <th><?php echo $this->Paginator->sort('Org.name');?></th>
+                <th><?php echo $this->Paginator->sort('Org.name', __('Organisation name'));?></th>
```

Comment:
Not sure this is correct, @iglocska could you comment?

Reply:
It's correct. By default it would "humanize" the name and just display it. Passing a second argument will simply force a display name, so it's cool.